src/codemagic/tools/xcode_project.py

```diff
@@ -111,6 +118,12 @@ def use_profiles(
         """
         from .keychain import Keychain
 
+        if isinstance(custom_export_options, CustomExportOptions):
+            custom_export_options = custom_export_options.value
+
+        print(custom_export_options)
+        return
```

Comment:
soooo, this return right here makes pretty much every build that uses `xcode-project use-profiles` fail 😅

Reply:
New version with a fix is incoming 🤦

Reply:
Fix is up with [v0.36.1](https://github.com/codemagic-ci-cd/cli-tools/releases/tag/v0.36.1).